### DISCUSSION TITLE
Initialize-Configuration

### DISCUSSION THREAD

**Discussion by jdhitsolutions** [SamErde/PSPreworkout#6] [General]
You might consider renaming `Initialize-Configuration`. The noun is extremely generic and could mean almost anything. Maybe something like `Initialize-PSPreWorkoutConfiguration`. A little long, but you have aliases and there's tab-completion. Now, there's no confusion about what this command will do and you avoid naming collisions from other authors who don't plan ahead. 😉

**Comment by SamErde** [comment]
GREAT point! I will absolutely rename this function. Since the concept is to configure my initial PowerShell and Git settings, I'll try to think of a name along the lines of `Initialize-PowerShellEnvironmentConfiguration` or `Set-PowerShellEnvironmentConfiguration`...but just a little shorter. 😊

**Reply by jdhitsolutions** [reply]
Don't worry about making it shorter. Make it meaningful. You can always create shorter aliases. But, I abbreviate things like `PowerShell` as `PS`.
